hack/verify-manifests.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
+
+set -o errexit
+set -o nounset
+set -o pipefail
+
+SCRIPT_ROOT=$(dirname "${BASH_SOURCE}")/..
+
+DIFFROOT="${SCRIPT_ROOT}/deploy/single/"
+
+cleanup() {
+  git checkout "${DIFFROOT}"
+}
+trap "cleanup" EXIT SIGINT
+
+cleanup
```

Comment:
this (`cleanup`) feels surprising if someone runs this command by hand. Maybe fail instead of trying to fix?

```suggestion
if ! git status --porcelain --untracked-files=no "$DIFFROOT" ; then
    echo "error: please run this script on a clean working copy"
    exit 1
fi

```